docs/modules/migrate/pages/dmt-connection-config.adoc

```diff
@@ -0,0 +1,174 @@
+
+=== Update Data Structures
+
+To update the data structure information, complete the following steps:
+
+. Navigate to the folder containing the extracted DMT package
+. Open the _migration_config/data/imap_names.txt_ and/or the _migration_config/data/replicated_map_names.txt_ file in your favorite editor
+. Update the file content to match the names of your maps
++
+If you have multiple data structures use a new line for each map name, or use wildcards to select multiple data structures using a single entry. For further information on using wildcards, see the xref:configuration:using-wildcards.adoc[Using Wildcards] topic.
+
+. Save the files
+
+==== ReplicatedMap Metadata Migration
+
+By default, Replicated Map metadata is migrated. However, for some versions you must add the provided extension JAR to your class path and restart the cluster before starting the migration to migrate Replicated map metadata. For further information on when you need to use the extension JAR and how to use it, see the xref:migrate:dmt-source-cluster-extension.adoc[] topic.
+
+If you want to disable migration of Replicated map metadata, open the _migration_config/migration_config.yaml_ file in your favorite editor and complete the following steps:
+
+. Set `migrateReplicatedMapMetadata` to `false`
+. Save the file
+
+==== CP Data Structures
+
+NOTE: For some versions you must add the provided extension JAR to your class path and restart the cluster before starting the migration to migrate CP data structures. For further information on when you need to use the extension JAR and how to use it, see the xref:migrate:dmt-source-cluster-extension.adoc[] topic.
+
+You can migrate the following CP data structures:
+
+* xref:data-structures:iatomiclong.adoc[]
+* xref:data-structures:iatomicreference.adoc[]
+* xref:data-structures:icountdownlatch.adoc[]
+
+If a CP data structure has no data, it is not migrated. For example, where the `set()` method has not been called on an existing `AtomicReference`. This is because such data structures are created lazily and do not exist in the cluster until used. This avoids the creation of a server-side resource with null data.
+
+You can update the following files with a list of the CP structures to migrate:
+
+** _migration_config/data/iatomic_long_names.txt_
+** _migration_config/data/iatomic_reference_names.txt_
+** _migration_config/data/icount_down_latch_names.txt_
+
+To configure the migration of CP data structures, you must update the _migration_config.yaml_ file located in the _migration_config_ folder.
+
+The CP data structure configuration options are described in the following table:
+
+[%header,cols="1,2"]
+|===
+|Option |Description
+
+|`"matchHidden"`
+|Defines whether to include maps with names that start with a double underscore ($$__$$) in the migration when a wildcard (`*`) is included in the `run` command. For example, Hazelcast uses some internal maps with names starting with $$__$$ such as $$__$$sql.catalog.
```

Comment:
```suggestion
|Defines whether to include data structures with names that start with a double underscore ($$__$$) in the migration when a wildcard (`*`) is included in the `run` command. For example, Hazelcast uses some internal maps with names starting with $$__$$ such as $$__$$sql.catalog.
```

This option is a generic option that can be used with any data structure migration. If there are other places, with this description, we should also update them

Reply:
Also, I believe we should mention "migration config" instead of "run" command to be more clear with our description.

Reply:
Plus, since this option is generic, I don't think we should mention in the CP structures section, maybe in "Update Data Structures" section above.

Reply:
For reference, this is my description of the config option in the dmt package: https://github.com/hazelcast/hazelcast-mono/blob/d12b08f726de9dff17f2e06ac5188e9f52109d0b/distribution/src/root/migration_config/migration_config.yaml#L3 

I added this description.